ReportingServicesTools/ReportingServicesTools.psd1

```diff
@@ -98,7 +98,9 @@
         'Set-PbiRsUrlReservation',
         'Write-RsCatalogItem',
         'Write-RsFolderContent',
-        'Write-RsRestCatalogItem'
+        'Write-RsRestCatalogItem',
+        'New-RsSubscription',
```

Comment:
Could you please move the New-RsSubscription and New-RsScheduleXml after New-RsRestSession and before New-RsWebServiceProxy?

Reply:
Done.